ComputerVision/SDK/AnalyzeImage/Program.cs

```diff
@@ -0,0 +1,230 @@
+﻿using Microsoft.Azure.CognitiveServices.Vision.ComputerVision;
+using Microsoft.Azure.CognitiveServices.Vision.ComputerVision.Models;
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace ImageAnalyze
+{
+    class Program
+    {
+        private const string subscriptionKey = "0123456789abcdef0123456789ABCDEF";  //change this with your subscription key
+        private const string remoteImageUrl = "https://github.com/harishkrishnav/cognitive-services-dotnet-sdk-samples/raw/master/ComputerVision/Images/sample3.png";
```

Comment:
I find it odd that the remote Url is defined here, while the localFile is defined inside the Main method. Please make it consistent and define both next to each other inside the Main method. It is also how you are doing it in the Rest cases, so that would make us consistent with those as well.